build.sbt

```diff
@@ -15,6 +15,8 @@ lazy val root = (project in file(".")).enablePlugins(PlayLibrary)
 libraryDependencies ++= Seq(
   "com.typesafe.play" %% "play" % PlayVersion,
   "org.springframework" % "spring-context" % SpringVersion,
+  "org.springframework.boot" % "spring-boot" % "1.4.7.RELEASE",
```

Comment:
1.5.16.RELEASE is the newest release compatible with Spring 4.3.x:

http://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter/1.5.16.RELEASE

Reply:
Why not to use Spring 5 together with SpringBoot 2.0?